x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_types/types.ts

```diff
@@ -227,37 +227,6 @@ export interface SignalSource {
   kibana?: SearchTypes;
 }
 
-export interface BulkItem {
```

Comment:
These types were made when they weren't available in the es client I think? they're available on the client now, so we can delete these